website/content/partials/api/restricted-endpoints.mdx

```diff
@@ -6,6 +6,7 @@
 
 API path                                    | Root | Admin
 ------------------------------------------- | ---- | -----
+`sys/activation-flags/.../activate`         | YES  | NO
```

Comment:
I wanted to confirm that we had an alert on the API docs as well, but I don't see the `sys/activation-flags` endpoint listed in the API docs, is it documented somewhere else?

Reply:
It isn't documented properly yet as a full endpoint in the API section yet.

I do think it should be added with other paths at some point, but I don't think it's absolutely necessary at this time. At this moment this path only relates to Secrets Sync. Since all of the places sync is used or documented mention how to activate the feature (including Vault CLI itself linking to the docs), I think it is okay to defer until we have a 2nd feature gated by these activation flags.

Do you think there is any issues with this approach?